<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>152</width>
      <height>293</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>串口设置</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>波特率(&amp;S)</string>
            </property>
            <property name="buddy">
             <cstring>BaudRate</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="BaudRate">
            <property name="editable">
             <bool>false</bool>
            </property>
            <property name="currentText">
             <string>请选择</string>
            </property>
            <property name="maxVisibleItems">
             <number>16</number>
            </property>
            <property name="insertPolicy">
             <enum>QComboBox::InsertAtCurrent</enum>
            </property>
            <property name="minimumContentsLength">
             <number>0</number>
            </property>
            <property name="duplicatesEnabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>请选择</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4800</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>19200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>38400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>57600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>115200</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>串口号</string>
            </property>
            <property name="buddy">
             <cstring>ComList</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="ComList">
            <property name="editable">
             <bool>false</bool>
            </property>
            <property name="currentText">
             <string>请选择</string>
            </property>
            <property name="maxVisibleItems">
             <number>16</number>
            </property>
            <property name="insertPolicy">
             <enum>QComboBox::InsertAtCurrent</enum>
            </property>
            <property name="minimumContentsLength">
             <number>0</number>
            </property>
            <property name="duplicatesEnabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>请选择</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>校验位</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>无校验</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>奇校验</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>偶校验</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>0校验</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1校验</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>数据位</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>停止位</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="comboBox_2">
            <item>
             <property name="text">
              <string>8位</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9位</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="comboBox_3">
            <item>
             <property name="text">
              <string>1 位</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1.5 位</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2 位</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="ComCtrl">
          <property name="text">
           <string>打开串口</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>绘图设置</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="TemOpen">
          <property name="text">
           <string>打开波形显示</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="TemCtrl">
          <property name="text">
           <string>暂停波形显示</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="draw" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>591</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionsave"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>draw</class>
   <extends>QWidget</extends>
   <header>draw.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
